lib/child_process.js

```diff
@@ -277,7 +271,7 @@ exports.execFile = function(file /*, args, options, callback*/) {
         if (!encoding)
           _stderr.push(chunk);
         else
-          _stderr += chunk;
+          _stderr += chunk.toString(encoding);
```

Comment:
Couldn't this yield incorrect results if a variable-width encoded character is split across two chunks?